### PR TITLE
Bills Page fixed

### PR DESCRIPTION
**Pull request by Lester-Fong**
Created form for Purchase Order and New Bill

### CODE REVIEW

File: resources/views/vendors/bills/forms/addBillModal.blade.php [hunk 1]
```diff
@@ -1,80 +1,153 @@
...
-            <input type="number" name="country" class="form-control">
-        </div>
-        <div class="col">
-            <label for="email">Order number:</label>
-            <input type="number" name="tinNum" class="form-control">
-        </div>
+        <label for="email" class="col-form-label col-lg-2">Bill number:</label>
+        <input type="number" name="country" class="form-control col-lg-4">
+        <label for="email" class="col-form-label col-lg-2">Order number:</label>
+        <input type="number" name="tinNum" class="form-control col-lg-4">
```
Comment (xyberpastoril) [inline resources/views/vendors/bills/forms/addBillModal.blade.php:26]: Both of these fields must be in the same column.

File: resources/views/vendors/bills/forms/purchaseOrderModal.blade.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    <div class="row form-group">
+        <label for="email" class="col-form-label col-lg-2">Telephone:</label>
+        <input type="number" name="fax" class="form-control col-lg-4">
+        <label for="email" class="col-form-label col-lg-2">Attachment:</label>
+        <div class="input-group col-md-2 col-lg-4">
+            <div class="custom-file">
+            <input type="file" class="custom-file-input" id="inputGroupFile03">
+            <label class="custom-file-label" for="inputGroupFile03">Choose file</label>
+            </div>
+        </div>
```
Comment (xyberpastoril) [inline resources/views/vendors/bills/forms/purchaseOrderModal.blade.php:37]: Keep this field at the bottom to be consistent with other pages.

File: resources/views/vendors/vendors/forms/addVendorModal.blade.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">FAX:</label>
+        <input type="text" name="fax" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Mobile number:</label>
+        <input type="text" name="mobile_num" class="form-control col-lg-4">
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Contact Person:</label>
+        <input type="text" name="contact_person" class="form-control col-lg-4">
```
Comment (xyberpastoril) [inline resources/views/vendors/vendors/forms/addVendorModal.blade.php:34]: Sir @justinmanigo , what field is linked to `Contact Person`? The `mobile number` field? I think this one needs to be grouped to minimize confusion.

**Review by xyberpastoril — COMMENTED** [review]
So far so good for me! 🚀

**Review by justinmanigo — APPROVED** [review]

**Pull request merged**